source/comply/audit-log.rst

```diff
@@ -26,7 +26,7 @@ Open ``config.json`` and navigate to the audit settings. The following `configur
 
     - Enable settings to write audit files locally
     - Specify file size
-    - Specify backup interval
+    - Specify the backup interval
     - Add compression
     - Set maximum age to manage file rotation
 
```

Comment:
This is a helpful change, thank you! I am curious as to why only this point got edited and not points two and five? Looking at this suggestion it seems like the same could apply to those. :)

Reply:
This is because backup intervals can be crucial so we are trying to give it the extra emphasis by supplying the definite article 'THE'. File Size can be of any amount, doesn't necessarily have to be a definite one. But still, there's no hard and first rule, so we can either leave it for the other two or else can add for them as well.

Reply:
Thank you - that makes sense. I'm happy with it as-is.